src/test/java/picard/util/help/DocumentationGenerationIntegrationTest.java

```diff
@@ -0,0 +1,79 @@
+package picard.util.help;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.ServiceLoader;
+import java.util.Set;
+import java.util.spi.ToolProvider;
+
+/**
+ * Smoke test to run doc gen on a subset of classes to make sure it doesn't regress.
+ */
+public class DocumentationGenerationIntegrationTest {
+    /**
+     * Entry point for manually running the picardDoc process on a subset of packages from within picard.
+     */
+    private static String[] docTestPackages = {
```

Comment:
We could probably fix this issue by re-arranging some things but lets not do it now.